### DISCUSSION TITLE
Portugues Translation

### DISCUSSION THREAD

**Discussion by manuelchaves** [OrcaSlicer/OrcaSlicer#1162] [Dev]
hello, I would like to help translate Orca into Portuguese BR, how do I proceed?

**Comment by SoftFever** [comment]
@manuelchaves 
Cool.
Let me generate an empty Portuguese .po file for you to work with.

**Comment by manuelchaves** [comment]
Ok, do I need some specific app or direct txt?  As a reference I have boards from Lerdge, INaV and others.  Free of charge, just as a thank you to the excellent Orca.  tks

**Reply by Rodrigobr6** [reply]
Olá, a tradução será introduzida no orca?

**Comment by SoftFever** [comment]
@manuelchaves 
I've create a branch for Portuguese (Brazilian) language. [
feature/pt_BR](https://github.com/SoftFever/OrcaSlicer/tree/feature/pt_BR)

The file to work with is https://github.com/SoftFever/OrcaSlicer/blob/feature/pt_BR/bbl/i18n/pt_BR/OrcaSlicer_pt_BR.po
I usually use a text editor to modify .po files, but I can imagine that translating a file from scratch would be a bit cumbersome.
 
@hliebscher, you seem to have more knowledge on this topic. Could you provide some advice on the available tools for editing .po files?

**Reply by YvesYuri** [reply]
I have a .po file in Portuguese that is 96% translated. How to upload to your I have .po file in Portuguese 96% translated. How to upload? I used PoEdit

**Reply by DvoraNoob** [reply]
I think u need to do a Pull Request

**Reply by Rodrigobr6** [reply]
Olá a tradução será introduzida no orca?

**Reply by izanette** [reply]
@YvesYuri , Do you still have this file? Maybe we can work together to finish the translation and put it on the next release.

**Comment by manuelchaves** [comment]
I really like POEdit https://poedit.net/download simple but with excellent features that help a more cohesive translation.  It also accepts xliff and json files

**Comment by SoftFever** [comment]
I should give POEdit a try next time

**Comment by SoftFever** [comment]
Moving it to discusstion/dev forum

**Comment by izanette** [comment]
What is the current state of this development? I'd like to help with the translation but couldn't find the feature/pt_BR branch nor the [OrcaSlicer_pt_BR.po](https://github.com/SoftFever/OrcaSlicer/blob/feature/pt_BR/bbl/i18n/pt_BR/OrcaSlicer_pt_BR.po) file.

**Comment by jcalmeida85** [comment]
for when pt-pt or pt-br ??

**Comment by Rodrigobr6** [comment]
@SoftFever When will the translation of Orca be made to PT-BR?

**Comment by jcalmeida85** [comment]
hello
here is the translation made by me, with small flaws but to be corrected by me or by the community
Based on the Spanish language version, just paste the orcaslicer.mo which is immediately functional.

just choose the Spanish language that automatically stays in Portuguese

I hope it will be fixed by the developers or integrated correctly in slicer

https://drive.google.com/drive/folders/1dglJaCzbU5fNjCXmhrLxYW8tUlaHcgnP?usp=drive_link

**Reply by ramorimdias** [reply]
Hello, I finished the translation for pt_BR. Thanks @jcalmeida85 for your work, it inspired me to start. I found it was easier to start from scratch as your file had a lot of misspelling and missing letters. I'm trying to push the file to the repository, but I am no expert in github, I don't know to do it. If anyone know how to do it so it can be on the next update it would be helpful. You can do it manually replacing the OrcaSlicer.mo file in C:\Program Files\OrcaSlicer\resources\i18n. 

here is the file: [https://1drv.ms/u/s!AkK6OpxBKxSVpH1JpoQvW50p-XLL?e=bW5xjF](url)

**Reply by jcalmeida85** [reply]
excellent work
as I said, it still had some errors and was in development but it ended up contributing to get even better.
at the moment I think it is the starting point to include the Portuguese language in a future update.
thanks for the community
@SoftFever now it's up to you ;)

**Reply by cesarapessoa** [reply]
Não consigo acessar o arquivo.

**Reply by DvoraNoob** [reply]
Link atualizado no telegram, estamos usando o Crowndin

**Reply by carlosreis1975** [reply]
nao consigo acessar o arquivo amigo

**Comment by DvoraNoob** [comment]
> Hello, I finished the translation for pt_BR. Thanks @jcalmeida85 for your work, it inspired me to start. I found it was easier to start from scratch as your file had a lot of misspelling and missing letters. I'm trying to push the file to the repository, but I am no expert in github, I don't know to do it. If anyone know how to do it so it can be on the next update it would be helpful. You can do it manually replacing the OrcaSlicer.mo file in C:\Program Files\OrcaSlicer\resources\i18n.
> 
> here is the file: [https://1drv.ms/u/s!AkK6OpxBKxSVpH1JpoQvW50p-XLL?e=bW5xjF](url)

Eu vou tentar buildar uma versão aqui para testar

English: I'll try to build a version to test

**Comment by ramorimdias** [comment]
For brazilians/portugueses I created a telegram link so we can chat freely and try to improve the translation. [https://t.me/+VMteJn3wrYY0NmZk](url) I have a friend of mine that is helping with some corrections. Sorry I don't know how to use github

**Reply by DvoraNoob** [reply]
I only use Telegram, and your po file has a lot of errors, I'll try fix it

**Reply by ramorimdias** [reply]
Hello, I'll put the last .po and .mo on this shared folder: [https://1drv.ms/f/s!AkK6OpxBKxSVpH44ENq5R6T_cxY6?e=2wILhA](url)
I'm still making some corrections on the translation to be more coherent.
ps: added a telegram group

**Reply by carlosreis1975** [reply]
poderia me enviar os arquivos amigo

**Reply by jcalmeida85** [reply]
ja podes utilizar apartir daqui ;
https://github.com/SoftFever/OrcaSlicer/releases/tag/nightly-builds

**Reply by carlosreis1975** [reply]
Thank you very much for your attention and help my friend

**Reply by carlosreis1975** [reply]
Let's create a group to translate Creality Print, friend?

**Reply by carlosreis1975** [reply]
unfortunately this latest version of yours is getting infected, take a look at us friend

**Reply by carlosreis1975** [reply]
omodo isntaller ta dando como infectado por trojan

**Reply by LemosBR** [reply]
Gostaria de baixar a tradução ou entrar no grupo do  telegram, mas todos os links que clico dão erro 404.

**Reply by DvoraNoob** [reply]
@LemosBR [Grupo Telegram](https://t.me/+VMteJn3wrYY0NmZk)

**Comment by Caminhaleonel** [comment]
Vou participar desse projeto Tambem. Entrando aqui no telegram

**Comment by DvoraNoob** [comment]
> omodo isntaller ta dando como infectado por trojan

Não tem nada infectado amigo, é normal navegador ou antivirus "ruim" avisar que tem algo errado, já que é uma versão de dev de um software opensource e está em constantes modificações e daí nunca fica como um app seguro, mas pode ficar tranquilo

**Reply by carlosreis1975** [reply]
ok amigo obrigado

**Reply by carlosreis1975** [reply]
o engracado que o alerta vem do instaler ja o portable esta tudo certo

**Reply by DvoraNoob** [reply]
Normal

**Reply by cidstn** [reply]
Como faço pra ajudar? Ainda precisam de ajuda ou já está ok? Tenho o arquivo do dia 01/05/24 atualizado na tradução, mas gostaria de ver o que vcs estão trabalhando. abs

**Reply by DvoraNoob** [reply]
@cidstn [Grupo Telegram](https://t.me/+VMteJn3wrYY0NmZk)

**Reply by Orion7123** [reply]
Faz sentido o que você explicou, realmente esses alertas muitas vezes são falsos positivos quando se trata de versões em desenvolvimento. Eu leio bastante coisa em fóruns assim e no meu tempo livre gosto de jogar em [https://mostbet1.com.pt/](https://mostbet1.com.pt/) é um site que curto bastante e sempre me diverte.

**Comment by Roger55555** [comment]
Tradução oficial em pt-br. Há alguma previsão? Pretendo testar OrcaSlicer, mas sem a tradução completa fica difícil. Entendo o trabalho e disponibilidade para tradução a ser feito e o necessário tempo para isto, portanto apenas gostaria de saber se tem alguma previsão.

**Reply by ramorimdias** [reply]
Olá, a tradução já está implementada nas versões de dev, vc pode baixar aqui https://github.com/SoftFever/OrcaSlicer/releases/tag/nightly-builds 

será incluída na próxima versão oficial! Ainda estamos ajeitando algumas frases, mas no geral está tudo ok.

**Comment by Roger55555** [comment]
Obrigado por responder, vou baixar a tradução e começar a testar o
OrcaSlicer.
Eu uso e gosto do PrusaSlicer, mas tenho ouvido falar bastante do Orca que
resolvi testar.

Em dom., 26 de mai. de 2024, 14:27, ramorimdias ***@***.***>
escreveu:
